lib/rex/parser/acunetix_nokogiri.rb

```diff
@@ -509,7 +509,7 @@ def report_web_site(url,&block)
       address = resolve_address(host)
       return unless address
       # If we didn't create the service, we don't care about the site
-      service_object = db.get_service @args[:workspace], address, "tcp", port
+      service_object = db.services(:workspace => @args[:workspace], :hosts => {address: address}, :proto => "tcp", :port => port).first
```

Comment:
I haven't looked into the specifics yet - from what I understand of the original PR, `proto` isn't supported by the services command? 🤔 

https://github.com/rapid7/metasploit-framework/pull/14143/files#r490233322

Reply:
You cannot sort by `:proto` but us still gets passed to the query:

see lines 157 and 159:
https://github.com/rapid7/metasploit-framework/blob/93290e1fa6e1a8fb81b1d1bbde8e3ca247fc4c91/lib/msf/core/db_manager/service.rb#L139-L162